tests/raid_test.py

```diff
@@ -425,6 +429,8 @@ def test_bulk_insert_transactions(fx_session):
             avatar_address = "0x01069aaf336e6aEE605a8A54D0734b43B62f8Fe4"
             amounts = [
                 ("RUNESTONE_FENRIR1", 560, 0),
```

Comment:
To use `ClaimItems` action, I think we should wrap this to `FAV__RUNESTONE_FENRIR1`.